### PR TITLE
83 define input output for tiltindicator and tiltindicatorafter onboarding

### PR DESCRIPTION
**Pull request by ysherstyuk**
Hi Sven,
All tables are generated! Let me know if you have any comments.

### CODE REVIEW

File: functions/spark_session.py
Comment (SKruthoff) [inline functions/spark_session.py:1]: The changes in this file should be deleted as they include the hotfix that is not relevant anymore

File: functions/processing.py [hunk 1]
```diff
@@ -426,83 +426,144 @@ def generate_table(table_name: str) -> None:
...
-                            .otherwise(F.lit(None)))   
+        df = read_table(spark_generate, 'ei_activities_overview_landingzone')
 
-        write_table(spark_generate, df, 'pstr_product_result_raw')
+        write_table(spark_generate, df, 'ei_activities_overview_raw')
 
-    # If the code is run as a workflow on databricks, we do not want to shutdown the spark session. 
-    # This will cause the cluster to be unusable for other spark processes
-    if not 'DATABRICKS_RUNTIME_VERSION' in os.environ:
-        spark_generate.stop()
```
Comment (SKruthoff) [inline functions/processing.py:508]: This should also be reversed to the newer state

**Review by SKruthoff — COMMENTED** [review]
I have the feeling that there is a lot of overlap with the other PRs, maybe good to discuss this on what the overlap is.

Also the parts of spark_session.py need to be removed as they are on an older version

**Review by SKruthoff — CHANGES_REQUESTED** [review]
I have the feeling that there is a lot of overlap with the other PRs, maybe good to discuss this on what the overlap is.

Also the parts of spark_session.py need to be removed as they are on an older version

File: functions/processing.py (hunk 1 above)
Comment (ysherstyuk) [reply functions/processing.py:508]: Yes we might need to disregard changes that you were making in other PRs as they are outdated. We still need to check what is needed for onboarding

**Pull request closed without merging**